### PR TITLE
Benchmark

### PR DESCRIPTION
**Pull request by Aadesha**

### CODE REVIEW

**Comment by mforets** [comment]
This PR adds some arithmetic benchmarks from [AGK18](https://easychair.org/publications/paper/9Tz3).

We plan to make a second PR to reproduce the Tables 3-6 in the same paper that uses some of the tests defined in the [Project Daisy](https://github.com/malyzajko/daisy) project.

**Comment by lbenet** [comment]
@Aadesha Thanks for your contribution. Is this ready for review?

**Comment by Aadesha** [comment]
It's ready, we are going to add another set of benchmarks too. Probably
tomorrow.

**Comment by mforets** [comment]
> It's ready, we are going to add another set of benchmarks too. Probably
tomorrow.

In a separate PR. This one is ready for review.

**Comment by lbenet** [comment]
For the last point of my comment, see https://github.com/JuliaDiffEq/DiffEqBenchmarks.jl/pull/32

**Comment by mforets** [comment]
> I think it is to have a Benchmark/README.md file,

+1   [This file](https://github.com/JuliaReach/LazySets.jl/blob/master/benchmark/benchmarks.jl#L2) has the basic commands to run and display results; @Aadesha could you collect them in `benchmark/README.md`?

> Rename Benchmarks directory to Benchmarks (note the final "s"), and perhaps use lower case. Maybe @dpsanders has something to say as a native speaker...

The filename and folder are the ones adopted by convention in `PkgBenchmarks`, however it should be `benchmark` (@Aadesha  mind the lower case), see [this doc](https://juliaci.github.io/PkgBenchmark.jl/stable/define_benchmarks.html#Defining-a-benchmark-suite-1).

> Add separately the scalar cases as well.

+1  IIUC, you'd like to have (in addition to the ones added here) `@benchmarkable $b1 + $b2` and so on.

> It is also a good idea to have an automatic way to create and display the (bechmark) results. I am thinking of using Weave.jl or Literate.jl. (Perhaps it is too much to ask for this PR, but well-worth the effort for maintenance.

That would be awesome; [export_markdown("results.md", results)](https://juliaci.github.io/PkgBenchmark.jl/stable/export_markdown.html) works but one has to manually plug that output into a document.

**Comment by mforets** [comment]
> Perhaps it is too much to ask for this PR

Yes, better leave that for later..

**Comment by Aadesha** [comment]
I agree with what all of you are saying,
I will take care of
1) README.
2) Lowercase 's'.
3) adding operations on scalar.
4)and Try for last one.

**Comment by lbenet** [comment]
The plan LGTM: address the first three points, and open an issue with the fourth one, so we don't forget about it.

**Comment by Aadesha** [comment]
That will be more appropriate, yes.

File: Benchmark/arithmetic.jl [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# ==========
+# Arithmetic
+# ==========
+
+SUITE["Arithmetic"] = BenchmarkGroup()
```
Comment (mforets) [inline Benchmark/arithmetic.jl]: Perhaps make two separate "suite", `SUITE["Arithmetic - scalar"]` and `SUITE["Arithmetic - vector"]` ?

**Comment by Aadesha** [comment]
PR is ready for review.

File: Benchmark/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+## How to run the benchmarks
+
+The files in this folder define a benchmark suite with the tools provided by [PkgBenchmark](https://github.com/JuliaCI/PkgBenchmark.jl) and [BenchmarkTools](https://github.com/JuliaCI/BenchmarkTools.jl).
+
+To run the benchmarks, execute:
+
+```julia
+julia> using PkgBenchmark
+
+julia> results = benchmarkpkg("TaylorModels.jl")
```
Comment (lbenet) [inline Benchmark/README.md]: I think here the command should be `benchmarkpkg("TaylorModels")`; at least, with the extension "jl" as it appears here, Julia threw an error when I tested.

**Comment by lbenet** [comment]
Thanks a lot! I'm merging

**Pull request merged**